README.md

```diff
@@ -45,6 +46,7 @@ Inputs are provided using the `with:` section of your workflow YML file.
 | key | Description | Nullable |
 | --- | --- | --- |
 | release_id | ID of the release built | true |
+| raw_version | Version of the release built | true |
```

Comment:
I know this value is called `raw_version` in the SDK but what do you think about just calling it `version` ? The raw part implies there is another version.

Reply:
Sure, works for me. I'll take the change.